Emerald.App/Emerald.App/Helpers/Settings/SettingsSystem.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Emerald.WinUI.Helpers.Settings.JSON;
+using Emerald.WinUI.Helpers.Settings.JSON;
```

Comment:
Emerald.WinUI namespace?

Reply:
It's not us, it's Microsoft. Let's just skip it and move on lol😅